cpp/modules/deck.gl/layers/src/line-layer/line-layer.h

```diff
@@ -82,7 +82,7 @@ class LineLayer::Props : public Layer::Props {
   std::function<ArrowMapper::FloatAccessor> getWidth{[](const Row&) { return 1.0; }};
 };
 
-struct alignas(lumagl::utils::kMinDynamicBufferOffsetAlignment) LineLayerUniforms {
+struct LineLayerUniforms {
```

Comment:
Do we feel confident that dropping AlignAs is OK here? Was it only needed when defining an Array?